### PR TITLE
fix: Update default details popup based on schedule item flags

### PR DESCRIPTION
**Pull request by apcro**
Fix: Modify the default schedule item details popup to respect the `isreadOnly` flag in the schedule item and not display the `edit` or `delete` actions.
Feat: Modify the default schedule item details popup to not display the `location` field if it is not set.
Feat: Modify the default schedule item details popup to not display the `atendees` field if it is not set.

### CODE REVIEW

**Comment by seonim-ryu** [comment]
Can one of the admins verify this patch?

File: src/js/view/template/popup/scheduleDetailPopup.hbs [hunk 1]
```diff
@@ -8,18 +8,21 @@
...
+        {{#if schedule.location}}<div class="{{CSS_PREFIX}}popup-detail-item"><span class="{{CSS_PREFIX}}icon {{CSS_PREFIX}}ic-location-b"></span><span class="{{CSS_PREFIX}}content">{{popupDetailLocation-tmpl schedule}}</span></div>{{/if}}
+        {{#if schedule.attendees}}<div class="{{CSS_PREFIX}}popup-detail-item"><span class="{{CSS_PREFIX}}icon {{CSS_PREFIX}}ic-user-b"></span><span class="{{CSS_PREFIX}}content">{{popupDetailUser-tmpl schedule}}</span></div>{{/if}}
         <div class="{{CSS_PREFIX}}popup-detail-item"><span class="{{CSS_PREFIX}}icon {{CSS_PREFIX}}ic-state-b"></span><span class="{{CSS_PREFIX}}content">{{popupDetailState-tmpl schedule}}</span></div>
         {{#if calendar}}
         <div class="{{CSS_PREFIX}}popup-detail-item"><span class="{{CSS_PREFIX}}icon {{CSS_PREFIX}}calendar-dot" style="background-color: {{schedule.bgColor}}"></span><span class="{{CSS_PREFIX}}content">{{calendar.name}}</span></div>
         {{/if}}
     </div>
+    {{#if schedule.isReadOnly}}
```
Comment (dongsik-yoo) [inline src/js/view/template/popup/scheduleDetailPopup.hbs]: How about using `unless` helper?

**Comment by dongsik-yoo** [comment]
@youjung-hong Please run this PR and check these display conditions of the default detail popup.

**Comment by youjung-hong** [comment]
@apcro Looks good to me! This is what we need. In some case, it looks like a bug!

Actually, there are some weird things on creation and detail popup's view model.

1) Creation/Update popup uses `schedule.attendee`, Detail popup uses `schedule.raw.creator`
2) Update popup uses `schedule.raw.location`, but schedule model put the location on `schedule.location`

So, after your PR is merged, default popup might not working as you will expect.
Congratulation on your first contribution!!

**Pull request merged**